src/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```diff
@@ -412,8 +412,8 @@
     <Compile Include="$(CommonPath)\Interop\FreeBSD\Interop.Process.cs">
       <Link>Common\Interop\FreeBSD\Interop.Process.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\Interop\Unix\System.Native\Interop.GetTimestamp.cs">
-      <Link>Common\Unix\System.Native\Interop.GetTimestamp.cs</Link>
+    <Compile Include="$(CommonPath)\Interop\Unix\System.Native\Interop.GetAbsoluteTime.cs">
```

Comment:
Is it needed here? It seems to be used by `Process.OSX.cs` only.

Reply:
Right, probably not.